ansible/roles/xroad-base/tasks/rhel.yml

```diff
@@ -3,6 +3,9 @@
   yum:
     name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
 
+- name: force yum to accept ca-key which signs repodata
```

Comment:
Is this necessary if eventually using `yum -y install xroad-securityserver-fi`?

Reply:
reworded to state purpose better